src/neo/NeoSystem.cs

```diff
@@ -119,7 +119,7 @@ public NeoSystem(ProtocolSettings settings, string storageEngine = null, string
         {
             this.Settings = settings;
             this.GenesisBlock = CreateGenesisBlock(settings);
-            this.storage_engine = storageEngine;
+            this.storage_engine = storageEngine ?? nameof(MemoryStore);
```

Comment:
Maybe we should log that the memory store is used

Reply:
What do you mean?

Reply:
The previous logic didn't do that. If do it, it should be done in neo-node.